docs/infra/test-platforms.md

```diff
@@ -0,0 +1,36 @@
+
+# Platform Testing Organization
```

Comment:
This needs more precise definition of Platform.

There are: 
- Architectures: x86, arm64, ...
- OSes: Windows, Linux, Linux-musl, macOS, Browser, ...
- OS flavors: Windows Client, Windows Server, Windows Nano, Azure Linux, Ubuntu, Debian, Alpine, Chrome Browser, Mozilla Browser, ...
- OS versions